uiTests/src/test/resources/ide.rhino.js

```diff
@@ -0,0 +1,56 @@
+
+importClass(java.lang.System);
+
+importClass(com.intellij.ide.GeneralSettings);
+importClass(com.intellij.ide.impl.ProjectUtil);
+importClass(com.intellij.openapi.application.ApplicationManager);
+importClass(com.intellij.openapi.application.ModalityState);
+importClass(com.intellij.openapi.progress.ProgressManager);
+importClass(com.intellij.ui.GuiUtils);
+
+// Do not run any of the methods in EDT.
+function Ide() {
+  this.configure = function () {
```

Comment:
A prime example of OOP made of _plywood and cardboard_ :trollface:

Reply:
:heart_decoration: